### PR TITLE
Update individual about us entry

### LINKED ISSUE #6: Update `AboutUs` page - Dominic

### PR DESCRIPTION
**Pull request by Arixeyeion**
Included myself into the about us page.

closes #6

### CODE REVIEW

**Review by KohGuanZeh — DISMISSED** [review]
LGTM

**Review by Javiery3889 — DISMISSED** [review]
LGTM!

**Review by Javiery3889 — APPROVED** [review]
LGTM

File: docs/images/Arixeyeion.png
Comment (Dethada) [inline docs/images/Arixeyeion.png:1]: Rename to `arixeyeion.png`, to match ``, currently the image is not loading

**Review by Dethada — APPROVED** [review]

**Review by KohGuanZeh — APPROVED** [review]

**Pull request merged**